config/spring19_seminar/bow_glove.conf

```diff
@@ -0,0 +1,35 @@
+// Import the defaults using the relative path
+include "../final.conf"
+
+
+// Output path
+project_dir = ${JIANT_PROJECT_PREFIX}
+
+
+// Optimization
+batch_size = 16
+dropout = 0.1 // following BERT paper
+lr = 2e-5  // following Jason, Alex
+max_epochs = 3 // If positive, maximum number of epochs
+val_interval = 100
+eval_val_interval = 100
+
+
+// Target tasks
+do_target_task_training = 0  // If true, after do_pretrain train the task-specific model parameters
+write_preds = "val,test"  // 0 for none, or comma-separated splits in {"train", "val", "test"} 
+                          // for which predictions are written to disk during do_full_eval
+
+
+// Pretraining tasks
+load_model = 0  // If true, restore from checkpoint when starting do_pretrain
+
+
+// Models
+// Model, bow + GloVe
```

Comment:
For some reason I also need bert_model_name = "" to make it work

Reply:
I also needed "allow_missing_task_map = 1"

Reply:
`bert_model_name = ""` is in `defaults.conf`